### ISSUE TITLE
Remove support for Siements BS2000

### ISSUE THREAD

**Issue by richsalz** [openssl/openssl#6232]
Look for Siemens and/or BS2000 in the code.

**Comment by dot-asm** [comment]
And MPE, Unixware, SCO, Nextstep. And why wait? Is there **any** indication whatsoever that they might work? Well, I can imagine some might still try Unixware, but the rest isn't simply there anymore... And as for Unixware, let them come forward with contributed .conf...

**Comment by richsalz** [comment]
Yes all those things.  It's kinda late in the release cycle to spend time worrying about those: we have many existing issues and PR's to address first.

**Comment by dot-asm** [comment]
Well, I'd say now is actually better moment that any other. Because it is now when system *maintainers*/integrators are evaluating it, and [expected] lack of feedback would actually be more meaningful. I mean between releases OpenSSL is more likely to be looked at by developers, and what they are more likely to care about are things working on just one computer, their, and it's highly unlike that their computer is running any of the OSes in question, straight impossible for BS2000 and MPE. And we want it to be viewed from another angle. Once again, this is actually because we expect **lack** of feedback...

**Comment by richsalz** [comment]
Maybe now is better.  But it's lower on the priority list then addressing the issues that we said we would.

**Comment by levitte** [comment]
Revisiting this (thanks @richsalz), I'm thinking along lines that we should probably remove old cruft that no one has touched for ages and don't seem to care much about today...  with some amount of prudence, but still.

The point is that it's fairly easy for anyone to create their own config files and use them, we have every bit of support present for that purpose (yes, it can still be made better, but...), and it wouldn't be too hard for someone to set up a wiki page with numerous examples (some of them migrated from our current 10-main.conf).

**Comment by levitte** [comment]
(I imagine that someone might create "config packages" with whatever's needed for their favourite platform too)

**Comment by paulidale** [comment]
I'm in agreement with removing old cruft.  3.0, as a major release, enables such.

**Comment by richsalz** [comment]
If #9209 #9208 #9204 and #9815 get merged (or even rejected), then this issue can be closed and the larger issue of sorting out platforms per the policy is left to the @openssl/omc

**Comment by paulidale** [comment]
Can this be closed?  The mentioned PRs are all closed.

**Comment by richsalz** [comment]
No it can't be closed.  `git grep --extended 'BS2000|OSD_POSIX'`
